update.sh

```diff
@@ -99,10 +107,11 @@ fi
 sudo nala upgrade
 
 # Install/Fix system files such as /etc/os-release
-cd ~
-mkdir ~/rhinoupdate/system-files/
-git clone https://github.com/rollingrhinoremix/assets ~/rhinoupdate/system-files/
-sudo mv ~/rhinoupdate/system-files/os-release /usr/lib/
+cd /usr/share/rhino
+mkdir /usr/share/rhino/rhinoupdate/system-files/
+git clone https://github.com/rollingrhinoremix/assets /usr/share/rhino/rhinoupdate/system-files/
+sudo mv /usr/share/rhino/rhinoupdate/system-files/os-release /usr/lib/
+chmod -R 775 /usr/share/rhino # In case this was ran with root - for regular users
```

Comment:
Will the script be required to run in root for it to operate? Is the command now going to be `sudo rhino-update`

Reply:
well basicly what happens is that if your a regular user, the `/usr/share/rhino` is read/write. If rhino-update is ran with sudo, then the files/directories that were added as sudo this would make it only writable to sudo, and this chmod should fix this for when the script is ran again.